App.tsx

```diff
@@ -24,23 +29,37 @@ export type StackParamList = {
 };
 
 const Stack = createStackNavigator<StackParamList>();
+const Drawer = createDrawerNavigator();
```

Comment:
The drawer navigator and photo stack should also have a type just like your Stack does on the line above.

That was part of the "Use Typescript to type your routes and params like we did in class." requirement.